README.md

```diff
@@ -1,2 +1,3 @@
 # git-demo
-to demonstrate how to use git and GitHub
+Arris Moise
```

Comment:
Could you please add a `-` so this becomes a list item?

Also, please make your name into a link that goes to that URL. The URL should not be visible.